src/test/swagger/oas3.yaml

```diff
@@ -307,7 +307,7 @@ paths:
   /pets:
     post:
       x-swagger-router-controller: TestController
-      operationId: updatePet
+      operationId: updatePetByType
```

Comment:
We were mapping two operations to the same controller implementation (`updatePet`) which was making it really confusing.